### ISSUE TITLE
Feature: labels concatenating values

### ISSUE THREAD

**Issue by jheeffer** [Gapminder/vizabi#1323]
A label now only supports óne constant/property/indicator.
Could be nice to have it support any number of property indicators in a self-defined string.

geo.name + ' in ' + geo.region + ' with a population of ' + population + ' in ' + year.

**Comment by angieskazka** [comment]
i don't see why you can't do it now
1. Define hooks for geo.name, geo.region and population
2. Time is defined separately
3. When in the tool-component, request values from the hooks
4. Code the label to construct a string from all the values you got

**Comment by jheeffer** [comment]
right now a hook is called after the thing it influences right? label, size, color?
so you would have 3 label hooks which are all passed to label?
I was thinking of keeping one hook (one hook = one 'thing') which requests multiple pieces of data.

**Comment by angieskazka** [comment]
no, the names of hooks are "color" and "size", but the fact that one hook refers to no more than one column in data is built deep in the design of hook. we would have to change a lot. 

i'd suggest go for 3 label hooks. and it makes sense! you hook your label to 3 different "things"

**Comment by jheeffer** [comment]
Yes it would mean changing a lot, but to me one hook per 'marker-property' (label/size/color/axis_x/axis_y) makes more sense. Having
```
marker {
  label1 { ... }
  label2 { ... }
  label3 { ... }
}
```
Looks to me like there's three different labels on marker.
